runbotics/runbotics-orchestrator-ui/src/src-app/Actions/sharepointExcel.actions.ts

```diff
@@ -173,49 +167,33 @@ const getSharepointExcelActions: () => Record<string, IBpmnAction> = () => ({
                         title: translate('Process.Details.Modeler.Actions.Common.Input'),
                         type: 'object',
                         properties: {
-                            content: {
-                                title: translate('Process.Details.Modeler.Actions.SharePointExcel.SetCell.Content'),
-                                type: 'string',
-                            },
                             cell: {
                                 title: translate('Process.Details.Modeler.Actions.SharePointExcel.SetCell.Cell'),
                                 type: 'string',
                             },
```

Comment:
you could add a pattern to validate the input (regex already exists: ActionRegex.EXCEL_CELL_ADDRESS), now if the user types only A the input is correct an the action is successfully completed. I think we should capture such incorrect input and throw an error